### ISSUE TITLE
[Desktop] Binance widget should be accessible via tab index

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#9256]
## Description 

Binance widget should be accessible via tab index

## Steps to Reproduce

   1. Connect Binance widget
   2. Open a new tab 
   2. Press tab to select the next element
   3. Binance widget isn't focused as there is no input filed in summary view

## Actual result:

Binance widget isn't accessible via tab index
## Expected result:
Binance widget should be accessible via tab index

## Reproduces how often: 

Easy 
## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? NA
- Can you reproduce this issue with the dev channel? NA 
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

**Comment by srirambv** [comment]
Verification passed on 

Brave | 1.11.77 Chromium: 83.0.4103.116 (Official Build) beta (64-bit)
-- | --
Revision | 8f0c18b4dca9b6699eb629be0f51810c24fb6428-refs/branch-heads/4103@{#716}
OS | Linux

- Verified Binance widget is accessible through tab index
- Logged #10508 as a follow up
